libs/ledger/include/ledger/chain/block.hpp

```diff
@@ -46,6 +46,8 @@ class Block
   using Slice    = std::vector<TransactionLayout>;
   using Slices   = std::vector<Slice>;
   using DAGEpoch = fetch::ledger::DAGEpoch;
+  using Hash     = Digest;
```

Comment:
Think that we should just use `Digest`

Reply:
Using the `Digest` name here will be a little problematic, since we already have publicly available type alias `Digest` defined in `fetch::ledger` namespace (declared in the `libs/ledger/include/ledger/chain/digest.hpp` file).

Reply:
I can simply remove the `Hash` declaration here.